cmd/minishift/cmd/util/util.go

```diff
@@ -160,3 +163,22 @@ func OcClusterDown(hostVm *host.Host) error {
 	_, err := sshCommander.SSHCommand(cmd)
 	return err
 }
+func GetOpenShiftVersionDuringStart() (string, error) {
```

Comment:
The name `GetOpenShiftVersionDuringStart` sounds weird to me as there is no co-relation between getting OpenShift versions from github and getting the versions during minishift start.

Reply:
@LalatenduMohanty any good name?

Reply:
@LalatenduMohanty  `GetDefaultOrLatestOpenShiftVersion`?

Reply:
@budhram GetLatestOpenShiftVersion sounds fine to me.